### PR TITLE
81 Fixed spinner when loading questions

### PR DESCRIPTION
**Pull request by ShevchenkoKirilo**
Now spinner always appears in the card while loading, spinner disappears if we get an error

### CODE REVIEW

**Comment by Anatolii-Ivanov** [comment]
We need spinner on opening one question, not for the whole page, please

**Comment by ShevchenkoKirilo** [comment]
image

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/lessons/lesson/lesson.component.sass [hunk 1]
```diff
@@ -250,3 +250,13 @@
...
+
+.spinner-container
+    position: relative
+    color: $white
+    height: 200px
+
+.spinner
+    position: absolute
+    top: 0
+    left: 49%
```
[the remark this answers is not in the document]
Comment (ShevchenkoKirilo) [reply frontend/src/app/modules/lessons/lesson/lesson.component.sass]: fixed

**Review by tatianahutii — DISMISSED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

File: frontend/src/app/modules/lessons/lesson/lesson.component.sass (hunk 1 above)
[the remark this answers is not in the document]
Comment (ShevchenkoKirilo) [reply frontend/src/app/modules/lessons/lesson/lesson.component.sass]: It makes big white space above spinner. Already tried this. Justify content and align items don't help

**Review by tatianahutii — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**